src/i18n/en.js

```diff
@@ -416,11 +417,24 @@ You can select **up to 10 Lists** to exclude them from the limit of Emails. Cont
       upgrade_plan_url: `${urlUpgradePlan}`,
     },
     select_lists: 'Select Lists',
-    subtitle_MD: `
+    shipments_quantity: {
+      subtitle_MD: `
 Set a **maximum number of Emails** your Contacts will be able to receive from you in a period of time.
 
 Want to know more? Press [HELP](${urlHelp}/contact-policy).`,
+      title: '(TBD)Quantity of shipments',
+    },
     success_msg: 'Changes have been saved.',
+    time_restriction: {
+      hour_from_aria_label: '(TBD)From Hour',
+      hour_to_aria_label: '(TBD)To Hour',
+      legend: '(TBD)Configure and set a range of shipments for your Campaigns. Choose when you want the shipment to be executed:',
+      time_slot_hour_from_label: '(TBD)From',
+      time_slot_hour_to_label: '(TBD)To',
+      time_slot_toggle_text: "(TBD)Only in a specific time slot considering your account's time zone.",
+      title: '(TBD)Time restriction',
+      weekdays_toggle_text: '(TBD)Only from Monday to Friday',
```

Comment:
I would simply define acceptable content and use it (without the _TBD_). 

There is going to be time to improve it before the final release.